apps/frontend/src/app/5_pages/ZeroPage/types.ts

```diff
@@ -0,0 +1,6 @@
+import { Decimal } from '@sovryn/utils';
+
+export interface ILiquityBaseParams {
```

Comment:
```suggestion
export type LiquityBaseParams {
```
Always use `type` if possible.